src/Raven.Server/Documents/PeriodicExport/PeriodicExportRunner.cs

```diff
@@ -235,7 +235,6 @@ private void TimerCallback(object fullExport)
 
                         var dataExporter = new SmugglerExporter(_database)
                         {
-                            DocumentsLimit = _exportLimit,
                             RevisionDocumentsLimit = _exportLimit,
```

Comment:
What is a RevisionDocumentsLimit? If this is equivalent to DocumentsLimit then we need to remove it also.